airbyte-integrations/connectors/destination-postgres-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -56,14 +56,14 @@
       },
       "ssl_mode": {
         "title": "SSL modes",
-        "description": "SSL connection modes. \n <b>disable</b> - Chose this mode to disable encryption of communication between Airbyte and destination database\n <b>allow</b> - Chose this mode to enable encryption only when required by the source database\n <b>prefer</b> - Chose this mode to allow unencrypted connection only if the source database does not support encryption\n <b>require</b> - Chose this mode to always require encryption. If the source database server does not support encryption, connection will fail\n  <b>verify-ca</b> - Chose this mode to always require encryption and to verify that the source database server has a valid SSL certificate\n  <b>verify-full</b> - This is the most secure mode. Chose this mode to always require encryption and to verify the identity of the source database server\n See more information - <a href=\"https://jdbc.postgresql.org/documentation/head/ssl-client.html\"> in the docs</a>.",
+        "description": "SSL connection modes. \n  See more information - <a href=\"https://jdbc.postgresql.org/documentation/head/ssl-client.html\"> in the docs</a>.",
         "type": "object",
         "order": 7,
         "oneOf": [
           {
             "title": "disable",
             "additionalProperties": false,
-            "description": "Disable SSL.",
+            "description": "Chose this mode to disable encryption of communication between Airbyte and destination database.",
```

Comment:
I think all of these descriptions should say `Choose this mode...` rather than `Chose this mode...`. Could we fix that in this PR?